dispatch/static/manager/src/js/pages/ItemIndexPage.js

```diff
@@ -83,8 +84,20 @@ export default class ListItemsPageComponent extends React.Component {
   }
 
   render() {
-    // Make the title start with uppercase letter
-    const titleString = this.props.typePlural.replace(/^\w/, m => m.toUpperCase())
+    const titleString = this.props.pageTitle
+      // Make the title start with uppercase letter
+      || this.props.typePlural.replace(/^\w/, m => m.toUpperCase())
```

Comment:
Not sure if I love this approach, just because it's hidden from whoever is using the component. I think I'd prefer to make the `pageTitle` prop required